tests/concurrent_hash_map/concurrent_hash_map.cpp

```diff
@@ -55,13 +57,126 @@ namespace nvobj = pmem::obj;
 namespace
 {
 
+class key_equal {
+public:
+	bool
+	operator()(const pmem::obj::experimental::string &str, int rhs)
+	{
+		return str == std::to_string(rhs);
+	}
+
+	bool
+	operator()(int lhs, const pmem::obj::experimental::string &str)
+	{
+		return str == std::to_string(lhs);
+	}
+
+	bool
+	operator()(const pmem::obj::experimental::string &str1,
+		   const pmem::obj::experimental::string &str2)
+	{
+		return str1 == str2;
+	}
```

Comment:
it will be removed once comparison methods for persistent string will be finished, right?
put XXX then